_examples/interactive_textinput/password/main.go

```diff
@@ -3,7 +3,11 @@ package main
 import "github.com/pterm/pterm"
 
 func main() {
-	result, _ := pterm.DefaultInteractiveTextInput.WithMask("*").Show("Enter your password")
+	result, _ := pterm.DefaultInteractiveTextInput.
+		//WithDefaultValue("asdasdads").
+		WithMultiLine(true).
+		//WithMask("*").
+		Show("Enter your password")
```

Comment:
Do we want this example changed? Maybe a new example with default value, as it looks like this should still be a password example. I don't think `WithDefaultValue` really works with `WithMask` either.

Reply:
We don't want this example changed, and it was actually left there by accident while I was testing something. 😊